addons/mail/static/src/components/activity/activity.xml

```diff
@@ -116,8 +116,8 @@
                     <t t-if="activityView.activity.canWrite">
                         <div name="tools" class="o_Activity_tools">
                             <t t-if="!activityView.fileUploaderView">
-                                <Popover position="'right'" title="MARK_DONE">
-                                    <button class="o_Activity_toolButton o_Activity_markDoneButton btn btn-link" t-att-title="MARK_DONE">
+                                <Popover position="'right'" title="activityView.markDoneText">
+                                    <button class="o_Activity_toolButton o_Activity_markDoneButton btn btn-link" t-att-title="activityView.markDoneText">
                                         <i class="fa fa-check"/> Mark Done
```

Comment:
technically this is also `markDoneText`, and the "title" on the button is kind of useless if the same text is directly in the DOM, but ok to ignore as it was already there before